letsencrypt/storage.py

```diff
@@ -588,6 +612,7 @@ def new_lineage(cls, lineagename, cert, privkey, chain,
                   cli_config.live_dir):
             if not os.path.exists(i):
                 os.makedirs(i, 0700)
+                logger.debug("Creating CLI config directory %s.", i)
```

Comment:
The name of the variable `cli_config` is a bit misleading (it's the CLI command line arguments/parsed configuration file).

Probably should just be `"Creating directory %s"`